dockerfiles/tasks.py

```diff
@@ -43,13 +43,15 @@ def down(c, volumes=False):
     'ext-theme': 'Enable new theme from ext-theme (default: False)',
     'scale-build': 'Add additional build instances (default: 1)',
     'ngrok': 'ngrok domain to serve the application. Example: "17b5-139-47-118-243.ngrok.io"',
+    'log-level': 'Logging level for the Django application (default: INFO)',
```

Comment:
Possible values are DEBUG, ERROR and WARNING, right?

Reply:
Yes. These are the supported values: https://docs.djangoproject.com/en/4.1/topics/logging/#loggers